src/components/EntityInfo.vue

```diff
@@ -192,6 +233,24 @@ export default class EntityInfo extends Mixins(EnumMixin) {
 
 #entity-info {
   background: $BCgovInputBG;
+
+  .breadcrumb {
+    padding: 0;
+  }
+
+  .v-breadcrumbs li {
+    font-size: .75rem;
+  }
+
+  ::v-deep {
```

Comment:
Yessss!

Reply:
The sonarCube doesn't like this tho.. :( It's a bug as far as it's concerned. Any ideas?

Reply:
I've seen this previously. I don't know how to disable this check so just ignore it for now. It won't report this again in future PRs.

Reply:
SonarCloud should be configurable... can you spend a small amount of time finding out how to configure it? Maybe Patrick knows...

Reply:
Great idea, when i finish this ticket i will 100% look into that 👍